apps/mobile/app/import-signers/index.tsx

```diff
@@ -2,60 +2,66 @@ import { SectionTitle } from '@/src/components/Title'
 import React from 'react'
 import { ScrollView } from 'react-native-gesture-handler'
 import Seed from '@/assets/images/seed.png'
+import Wallet from '@/assets/images/wallet.png'
+import Metamask from '@/assets/images/metamask.png'
 import { SafeFontIcon } from '@/src/components/SafeFontIcon'
 import { useScrollableHeader } from '@/src/navigation/useScrollableHeader'
 import { NavBarTitle } from '@/src/components/Title'
 import { SafeAreaView } from 'react-native-safe-area-context'
 import { SafeCard } from '@/src/components/SafeCard'
+import { router } from 'expo-router'
 
-/**
- * TODO: Add the images for each items
- * waiting for the design to exprot - figma is crazy
- */
 const items = [
   {
     name: 'seed',
     title: 'Import seed phrase or a private key',
     description: 'Enter a private key or a 12-24 word seed phrase.',
     icon: <SafeFontIcon name="wallet" size={16} />,
     Image: Seed,
+    imageProps: { marginBottom: -31 },
```

Comment:
why is it -31 here and -48 for the next image. Is this going to look ok on a smaller, bigger display? I guess you are adding those because of the safe area?